web-app/misc/swagger.json

```diff
@@ -3066,9 +3066,16 @@
           {
             "in": "query",
```

Comment:
Looks like the issue was that the frontend is now passing the parameters in the request body (judging from the code changes), not as query params, right? 

The `swagger.json` should reflect this change by changing the `in` field to `body` (if I remember correctly).

Reply:
> the frontend is now passing the parameters in the request body

Very curious, @juhah do you recall why this change from query params to request body? https://github.com/streamr-dev/streamr-platform/commit/b52775d1b4#diff-c245c1fe624a2e09cee0cbe7e228fa1aL515 Seems like this never should have worked, maybe you were working off a WIP branch or something in https://github.com/streamr-dev/streamr-platform/pull/408 We should get all the endpoints the frontend is using under test.

After this change, does that mean that if someone hits the endpoint with query params instead then they'll get the `NullPointerException` `500`?

Backend should probably do some actual validation of request data e.g. "date parameter missing from {request body|query parameters}" and responding with a `4xx` error. Grails/Spring likely provides some standardised tools for doing exactly this.

Reply:
Let’s keep the body params, it’s actually better in line with the other endpoints.

I doubt anyone is calling this endpoint apart from our own UI. Ok to make the breaking (lol, actually fixing) change.